run-qa-checks

```diff
@@ -15,14 +15,10 @@ rm -rf ./openwisp_controller/geo/tests/.pytest_cache \
 jshint openwisp_controller/pki/static/admin/pki/js/*.js
 jshint openwisp_controller/config/static/config/js/*.js
 jshint openwisp_controller/connection/static/connection/js/*.js
-stylelint "openwisp_controller/config/static/config/css/*.css"
-stylelint "openwisp_controller/connection/static/connection/css/*.css"
-
-
-# CSS checks
-stylelint "openwisp_controller/connection/static/connection/css/*.css"
```

Comment:
Why were those lines removed? :thinking:

Reply:
Ig this checks failing because of those changes

Reply:
Stylelint checks are successful, Jslint checks are failing.